addon/initializers/m3-store.js

```diff
@@ -56,6 +59,18 @@ export function extendStore(Store) {
       return this._super(modelName, id, clientId, storeWrapper);
     },
 
+    /**
+     * A thin wrapper around the API response that knows how to look up relationships.
+     *
+     * @param {string} url The URL path to query.
+     * @param {Object} options
+     * @param {string} [options.method=GET] The HTTP method to use.
+     * @param {Object} [options.params] The parameters to include
+     * @param {string} [options.cacheKey] A string to uniquely identify this request.
+     * @param {boolean} [options.reload=false]
+     * @param {boolean} [options.backgroundReload=false]
+     * @returns {Promise}
```

Comment:
It would be great to add the type of the resolution value as well.

maybe something like
```js
@returns {Promise<M3Model|RecordArray,Error>} a promise for loading `url` that fulfills to an `M3Model` if the response is a single resource or a `RecordArray` of `M3Model`s if the response is an array of resources.
```